### PR TITLE
refs #10970 - fix mass-assignment error

### PR DESCRIPTION
**Pull request by komidore64**

### CODE REVIEW

**Comment by johnpmitsch** [comment]
Fixed the problem for me! ACK

File: db/seeds.rb [hunk 1]
```diff
@@ -47,7 +47,15 @@ def format_errors(model = nil)
...
-  ProvisioningTemplate.find_or_create_by(:name => template["name"]).update_attributes(defaults.merge(template))
+  pt = ProvisioningTemplate.find_or_create_by(:name => template["name"])
+  defaults.merge(template).each do |k, v|
+    if k == :template_kind
+      pt.template_kind = v
+      next
+    end
+    pt[k] = v
+  end
+  pt.save!
```
Comment (jlsherrill) [inline db/seeds.rb]: why not just set the 3 attributes in the defaults hash?

Comment (jlsherrill) [inline db/seeds.rb]: I mean:
```
pt.vendoer = 'Katello'
pt.default = true
pt.locked = true
```

It seems quite a bit more straight forward than this.

**Comment by daviddavis** [comment]
I probably would wait on fixing mass assignment for Foreman models until after this PR is merged. It should fix this as well as a bunch of other places where we create/update models.

https://github.com/dLobatog/foreman/pull/13

File: db/seeds.rb (hunk 1 above)
Comment (komidore64) [inline db/seeds.rb]: hrm, i guess that certainly would be simpler. :)

**Comment by komidore64** [comment]
@daviddavis do you happen to know a time frame on dLobatog/foreman#13?

**Comment by johnpmitsch** [comment]
@komidore64 can we check what happens on a local checkout of that PR?

**Comment by komidore64** [comment]
@johnpmitsch sure thing. i'll check out that branch with katello `rails4` and see what happens.

**Comment by johnpmitsch** [comment]
ACK from me

**Comment by jlsherrill** [comment]
APJ

**Comment by johnpmitsch** [comment]
@jlsherrill jenkins wont pass as this is rails4 branch :)

**Pull request merged**